### ISSUE TITLE
location of directory for packages frequently gets lost or confused by IDE

### ISSUE THREAD

**Issue by mpbean** [forio/julia-studio#108]
Loading a package into directory will work but often typing "Using [package]" in the console will result in an error message: 
could not open file /Users/MB/Dropbox/MPB/Julia/[package]

My best guess is that this occurs after using File > Save for saving a Julia document in my working directory. Perhaps the Julia package library directory location is being changed along with the working directory?

QA could work to replicate the problem. First step will be do have different directories for the packages and the working directory. Then try loading and using packages. Then try writing and saving local Julia documents. Then try loading and using packages again until you get an error message.

**Comment by kbrydges** [comment]
Hi Michael-
   I can see if I can reproduce this and let Wes know what I find. I'll take a look today.

-Kat

**Comment by WestleyArgentum** [comment]
Thanks Kat, let me know if you have any trouble

**Comment by kbrydges** [comment]
Yes, I was able to reproduce this by first opening a file in Julia. Then, from the console, I tried to open one of the packages available to me using the "using" command and I got the same error message. It does look like it is only looking in the directory in which the file was opened from. 
julia> using TOML
could not open file /Users/gnandigam/Documents/Julia-test/Quad/TOML.jl

I am able to open the package if doubleclick on it from the "Package" section. 

Let me know if you want me to do anymore testing on this.
